apps/ensindexer/src/config/app-config.ts

```diff
@@ -96,23 +98,32 @@ export function getConfig() {
  * @returns the rate limit in requests per second (rps)
  */
 export const rpcMaxRequestsPerSecond = (chainId: number): number => {
-  const config = getConfig();
-  if (!config.chains[chainId]?.rpcMaxRequestsPerSecond) {
-    throw new Error(`RPC max requests per second not found for chain ID ${chainId}`);
+  const chainConfig = config.indexedChains[chainId];
```

Comment:
this can be simplified to `return config.indexedChains[chainId]?.rpcMaxRequestsPerSecond || DEFAULT_RPC_RATE_LIMIT`